WinUIGallery/ControlPages/InfoBadgePage.xaml

```diff
@@ -67,9 +67,11 @@
         <local:ControlExample x:Name="Example2" HeaderText="Different InfoBadge Styles" HorizontalContentAlignment="Stretch" >
             <local:ControlExample.Example>
                 <StackPanel Orientation="Horizontal" contract5Present:Spacing="20" HorizontalAlignment="Center">
-                    <InfoBadge x:Name="infoBadge2" Style="{StaticResource AttentionIconInfoBadgeStyle}" HorizontalAlignment="Right"/>
+                    <InfoBadge x:Name="infoBadge2" Style="{StaticResource AttentionIconInfoBadgeStyle}" HorizontalAlignment="Right"
```

Comment:
The xaml here does not match the xaml above (and it didn't before the change :)).